osmaxx/excerptexport/models/export.py

```diff
@@ -65,3 +76,23 @@ def _get_export_status_changed_message(self):
             'job_progress/messages/export_status_changed.txt',
             context=view_context,
         ).strip()
+
+    def _fetch_result_file(self):
+        from osmaxx.api_client import ConversionApiClient
+        from . import OutputFile
+        api_client = ConversionApiClient()
+        file_content = api_client.get_result_file(self.conversion_service_job_id)
+        file_location = get_default_private_storage().save(
+            name=str(uuid.uuid4()),
```

Comment:
I think improving it on the backend would be better, having the naming at one place only seems more sensible to me.